pkg/controller/webhooks/xobjectbuckets.go

```diff
@@ -55,6 +56,15 @@ func (p *XObjectbucketDeletionProtectionHandler) ValidateDelete(ctx context.Cont
 		return nil, fmt.Errorf("object is not valid")
 	}
 
+	creationTimestamp := bucket.GetCreationTimestamp()
+
+	now := time.Now()
+	age := now.Sub(creationTimestamp.Time)
+
+	if age < 61*time.Minute {
+		return nil, fmt.Errorf("XObjectBucket is too young to be deleted")
```

Comment:
NIT: Maybe print how many minutes remaining until it should be deleted.

Reply:
sure

Reply:
the phrasing "too young" really does not help. Either print the minutes as suggested above or just something like "XObjectBucket has been created less than 1 hour".